capella2polarion/converters/document_renderer.py

```diff
@@ -576,11 +575,25 @@ def _render_full_authority_documents(
 
                     project_data.new_docs.append(document_data)
 
-    def _extract_section_areas(self, html_elements: list[etree._Element]):
+    def _extract_section_areas(
+        self, html_elements: list[etree._Element], session: RenderingSession
+    ):
         section_areas = {}
         current_area_id = None
         current_area_start = None
         for element_index, element in enumerate(html_elements):
+            if (
+                current_area_id is None
+                and element.tag == "div"
+                and (
+                    matches := polarion_html_helper.wi_id_regex.match(
```

Comment:
Can we change the global patterns from here: https://github.com/DSD-DBS/capella-polarion/blob/9c80b3dc02fc660412df62d3a72c98bfd81f6d1f/capella2polarion/converters/polarion_html_helper.py#L15-L17

to be uppercase? Then it is consistent with the other global variables.

Reply:
Yeah, by now we used them only locally, I believe. But now we should adjust it